src/main/java/net/minecraftforge/event/TickEvent.java

```diff
@@ -60,10 +78,25 @@ public ClientTickEvent(Phase phase)
 
     public static class WorldTickEvent extends TickEvent {
         public final Level world;
-        public WorldTickEvent(LogicalSide side, Phase phase, Level world)
+        private final BooleanSupplier haveTime;
+        public WorldTickEvent(LogicalSide side, Phase phase, Level world, BooleanSupplier haveTime)
         {
             super(Type.WORLD, side, phase);
             this.world = world;
+            this.haveTime = haveTime;
+        }
+
+        /**
+         * Returns whether the server has enough time to perform any
+         * additional tasks (usually IO related) during the current tick.
+         * 
+         * @return <code>true</code> if there is time, <code>false</code> otherwise
+         * 
+         * @see {@link ServerTickEvent#haveTime()}.
```

Comment:
No need for that `{@link}` tag usage:
```suggestion
         * @see ServerTickEvent#haveTime()
```